src/components/schedule-visualizer/tests/__snapshots__/waypoint.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Renders correctly 1`] = `
 <svg>
-  <Waypoint
+  <Memo()
```

Comment:
This looks weird.

Here those guys where getting `<Component/>` instead of their component. https://github.com/facebook/jest/issues/8122

Reply:
Yeah, seems like some bug either in enzyme or jest, using memo also sometimes causes enzyme to find a component twice so I had to switch to the css selector.